### PR TITLE
add zh-TW translation

### PR DESCRIPTION
**Pull request by yyc1217**

### CODE REVIEW

File: src/i18n/zh-TW.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+(function(jsGrid) {
+
+    jsGrid.locales["zh-TW"] = {
```
Comment (tabalinas) [inline src/i18n/zh-TW.js]: "zh-tw" (lowercase)

File: src/i18n/zh-TW.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                cancelEditButtonTooltip: "取消編輯"
+            }
+        },
+
+        validators: {
+            required: { message: "欄位必填" },
+            rangeLength: { message: "欄位字串長度超出範圍" },
+            minLength: { message: "欄位字串長度太短" },
+            maxLength: { message: "欄位字串長度太長" },
+            pattern: { message: "欄位字串不符合Regular Expression規則" },
```
Comment (tabalinas) [inline src/i18n/zh-TW.js]: Is it on purpose that "Regular Expression" left in English?

**Comment by tabalinas** [comment]
Thank you for the PR! Could you please address a couple of comments before merging?

File: src/i18n/zh-TW.js (hunk 1 above)
Comment (yyc1217) [inline src/i18n/zh-TW.js]: Should I change file name to zh-tw.js too?

Comment (tabalinas) [inline src/i18n/zh-TW.js]: Yes, please.
I'm gonna fix it for "zh-cn" too.

File: src/i18n/zh-TW.js (hunk 2 above)
Comment (yyc1217) [inline src/i18n/zh-TW.js]: I will remove it

**Comment by yyc1217** [comment]
@tabalinas fixed.

**Comment by tabalinas** [comment]
Thank you!

**Pull request merged**